examples/MagicWeather/app/src/main/java/com/revenuecat/sample/ui/paywall/PaywallFragment.kt

```diff
@@ -31,8 +34,15 @@ class PaywallFragment : Fragment() {
         linearLayoutManager.orientation = LinearLayoutManager.VERTICAL
         recyclerView.layoutManager = linearLayoutManager
 
-        adapter = PaywallAdapter(null, didChoosePackage = { item: Package ->
-            purchasePackage(item)
+        adapter = PaywallAdapter(null, didChoosePaywallItem = { item: PaywallItem ->
```

Comment:
i wonder if we could have a boolean of some sort in this fragment like `showAllSubscriptionOptions` or, the opposite `useDefaultOptions`. then the user understands the different complexities of paywall setup. 

or just a comment of some sort on the different options.